src/pages/settings/Profile/DisplayNamePage.tsx

```diff
@@ -111,7 +111,7 @@ function DisplayNamePage({currentUserPersonalDetails}: DisplayNamePageProps) {
                             role={CONST.ROLE.PRESENTATION}
                             defaultValue={currentUserDetails.lastName ?? ''}
                             spellCheck={false}
-                            isMarkdownEnabled
+                            autoCapitalize="words"
```

Comment:
@rezkiy37 I think we should add the same fix to LegalNamePgae for consistency

Reply:
It definetely makes sense.